addons/tags/tags.php

```diff
@@ -147,6 +147,7 @@ public function tags_page() {
 		$tag_args['fields'] = 'count';
 		$found_query        = new \WP_Term_Query( $tag_args );
 		$tags_rows_found    = $found_query->get_terms();
+		$tags_rows_found    = ! is_array( $tags_rows_found ) ? (array) $tags_rows_found : $tags_rows_found;
```

Comment:
@mebishalnapit Note that `get_terms()` will return a numeric string. So you don't have to type cast it to an array. You should instead check `is_numeric`.